### PR TITLE
feat: add translations and separate chips

### PR DESCRIPTION
**Pull request by Olensavchenko**

### CODE REVIEW

File: packages/components/table/components/xm-table-filter-chips.component.ts [hunk 1]
```diff
@@ -149,6 +149,9 @@ export interface XmTableFilterInlineFilter {
         MatIconModule,
         MatMenuModule,
         XmTableFilterChipsControlComponent,
+        XmTranslationModule,
```
Comment (denisogolubev) [inline packages/components/table/components/xm-table-filter-chips.component.ts:152]: 3 times duplicated import

File: packages/components/table/components/xm-table-filter-chips.component.ts [hunk 2]
```diff
@@ -193,7 +196,11 @@ export class XmTableFilterChipsComponent {
 
     public remove(filter: XmTableFilterInlineFilter): void {
         const copy = cloneDeep(this.value);
-        delete copy[filter.name];
+        if(isArray(copy[filter.name])){
+            copy[filter.name] = (copy[filter.name] as any[]).filter(value => value !== filter.value)
```
Comment (denisogolubev) [inline packages/components/table/components/xm-table-filter-chips.component.ts:200]: According to the type you specified in the interface `FiltersControlValue` here should be `Primitive[]` not `any[]`

File: packages/components/table/components/xm-table-filter-chips.component.ts [hunk 3]
```diff
@@ -238,19 +245,34 @@ export class XmTableFilterChipsComponent {
                     view: overrideView as XmInlineControlDynamicView<unknown, unknown>,
                     edit: config as XmInlineControlDynamic<unknown>,
                 };
+                if(isArray(this.value[config.name])){
+                    return (this.value[config.name] as any[]).map((value, index) => {
```
Comment (denisogolubev) [inline packages/components/table/components/xm-table-filter-chips.component.ts:249]: According to the type you specified in the interface FiltersControlValue here should be Primitive[] not any[]

File: packages/components/table/components/xm-table-filter-chips.component.ts [hunk 4]
```diff
@@ -238,19 +245,34 @@ export class XmTableFilterChipsComponent {
...
+                        let title;
+                        if(config['title']) {
+                            title = config['title'][value] ? config['title'][value] : config.name;
+                        } else {
+                            title = config['title'] || config.name;
+                        }
+                        return {
```
Comment (denisogolubev) [inline packages/components/table/components/xm-table-filter-chips.component.ts:256]: if the key and value have the same name you can write only the value, then this `return` will be simpler, but do the same thing:
```ts
return {
     config,
     inlineConfig,
     value,
     removable: !config?.removable,
     title,
     name: config.name,
 } as XmTableFilterInlineFilter;
```

File: packages/components/table/components/xm-table-filter-chips.component.ts [hunk 5]
```diff
@@ -238,19 +245,34 @@ export class XmTableFilterChipsComponent {
...
+                            inlineConfig: inlineConfig,
+                            value: value,
+                            removable: config?.removable !== false,
+                            title: title,
+                            name: config.name,
+                        } as XmTableFilterInlineFilter;
+                    })
+                }
+
                 return {
```
Comment (denisogolubev) [inline packages/components/table/components/xm-table-filter-chips.component.ts:267]: also, this second return looks the same as the previous one, with some differences.

To not duplicate the same rows you can create the variable where will be stored initial data (duplicated one)
```ts
 const tableFilterInlineFilter: XmTableFilterInlineFilter = {
     config,
     inlineConfig,
     removable: !config?.removable,
     name: config.name,
 }
// then the first return will be like this
 return {
     ...tableFilterInlineFilter,
     value,
     title
 }

// and the second return will be like this
 return {
     ...tableFilterInlineFilter,
     value: this.value[config.name],
     title: config['title'] || this.value[config.name],
 }
```

in this way less duplicated code, and you can easily make changes for both of these returns.
But, these changes are up to you. Just an alternative way to write the code.

**Review by LevBD — APPROVED** [review]

**Review by denisogolubev — APPROVED** [review]

**Review by Sumragen — APPROVED** [review]

**Pull request merged**